Sources/AppStoreConnectCLI/Commands/TestFlight/Sync/TestFlightPullCommand.swift

```diff
@@ -34,8 +34,7 @@ struct TestFlightPullCommand: CommonParsableCommand {
         let testers = try service.listBetaTesters(filterIdentifiers: identifiers, limit: 200)
         let groups = try service.listBetaGroups(filterIdentifiers: identifiers)
 
-        let processor = TestflightConfigurationProcessor(appsFolderPath: outputPath)
-        try processor.writeConfiguration(apps: apps, testers: testers, groups: groups)
+        try writeConfiguration(apps: apps, testers: testers, groups: groups, to: outputPath)
```

Comment:
how about we add the Module name before global functions, this looks a bit clear to me.
```suggestion
        try FileSystem.writeConfiguration(apps: apps, testers: testers, groups: groups, to: outputPath)
```